libcontainer/seccomp/seccomp_linux.go

```diff
@@ -113,8 +113,8 @@ func InitSeccomp(config *configs.Seccomp) (int, error) {
 // Convert Libcontainer Action to Libseccomp ScmpAction
 func getAction(act configs.Action, errnoRet *uint) (libseccomp.ScmpAction, error) {
 	switch act {
-	case configs.Kill:
-		return libseccomp.ActKill, nil
+	case configs.Kill, configs.KillThread:
```

Comment:
Wondering if one of `configs.Kill` also should be deprecated on this side 🤔  (not for this PR, just thinking out loud?)

Reply:
We have the same problem in libcontainer/config -- Kill and KillThread are two distinct constants. Guess it makes sense to deprecate the former.

Reply:
Yup; that was my thought as well

I'm ok doing that separately, but something we should probably do

Reply:
Oh well, it's a bit bigger than I thought; see https://github.com/opencontainers/runc/pull/3466

@crazy-max feel free to cherry-pick that commit to here.

I also think we should really deprecate ACT_KILL, but it should be done in the runtime-spec repo first.

Reply:
@kolyshkin

> feel free to cherry-pick that commit to here.

done